sys-libs/tree-standalone/tree-standalone-1.0.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Compatibility library for musl"
+HOMEPAGE="https://github.com/void-linux/void-packages/tree/master/srcpkgs/musl-legacy-compat"
+
+LICENSE="|| ( BSD-2 BSD )"
+SLOT="0"
+KEYWORDS="~amd64"
+SRC_URI="https://raw.githubusercontent.com/void-linux/void-packages/master/srcpkgs/musl-legacy-compat/files/tree.h"
```

Comment:
SRC_URI should be directly under HOMEPAGE

Reply:
also, didn't we agree not to do this? :)

Reply:
yes, but I just pushed what I had. Will start fixing things now